### PR TITLE
add connection library / add mobx connection service / create Settings / refactoring

### PR DESCRIPTION
**Pull request by igor90007**

### CODE REVIEW

File: src/config/Settings.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import credentialsData from 'src/credentials.json'
```
Comment (ddzirt) [inline src/config/Settings.ts]: This is duplicated in at least 2 more RPs.

File: src/stores/domain/general.ts [hunk 2]
```diff
@@ -1,37 +1,57 @@
...
+import { Platform } from 'react-native'
 
-import InputsModel from 'src/models/InputsModel'
+import { ILanguageData, InputsModel } from 'src/models/InputsModel'
 
-interface Data {
-  [key: string]: string
-  languageSource: string
-  languageTranslated: string
+const checkConnectivity = () => {
```
Comment (ddzirt) [inline src/stores/domain/general.ts]: And where is this logic used? No meaningful usage whatsoever.

Comment (igor90007) [reply src/stores/domain/general.ts]: Annotation 2019-05-14 214049

File: src/config/Settings.ts (hunk 1 above)
Comment (igor90007) [reply src/config/Settings.ts]: you're right

File: src/stores/domain/general.ts (hunk 2 above)
Comment (ddzirt) [reply src/stores/domain/general.ts]: And wtf is this crap? Why is it not connected to the application life cycle?

**Pull request merged**